framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -981,60 +1068,76 @@ protected boolean visitIntersectionSubtype(
    * @param supertype a type that might be a supertype
    * @return true if {@code subtype} is a subtype of {@code supertype}
    */
-  protected boolean visitTypevarSupertype(
+  protected boolean visitType_Typevar(
       AnnotatedTypeMirror subtype, AnnotatedTypeVariable supertype) {
     return isSubtypeCaching(subtype, supertype.getLowerBound());
   }
 
   /**
-   * A type variable is a subtype if its upper bounds is below the supertype. Note: When comparing
-   * two type variables this method and visitTypevarSupertype will combine to isValid the subtypes
+   * A type variable is a subtype if its upper bound is below the supertype. Note: When comparing
+   * two type variables, this method and visitType_Typevar will combine to isValid the subtypes
```

Comment:
I don't understand what "will combine to isValid" means.  Could you reword?

Reply:
I don't understand this comment either.  I deleted it.